greenheart/simulation/greenheart_simulation.py

```diff
@@ -187,7 +187,7 @@ def run_simulation(config: GreenHeartSimulationConfig):
             config.orbit_config["plant"].update(
                 {"turbine_spacing": config.greenheart_config["site"]["wind_layout"]["turbine_spacing"]}
             )
-            warnings.warn(f"site depth in the orbit_config was {config.orbit_config['plant']['turbine_spacing']}, but 'turbine_spacing' in" 
+            warnings.warn(f"`turbine_spacing` in the orbit_config was {config.orbit_config['plant']['turbine_spacing']}, but 'turbine_spacing' in" 
```

Comment:
I think we just need to pick one and go with it. Numpy actually uses both backticks and single quotes in instances like this (error/warning messages) to set apart variable names. I'm inclined to leave it for now since I prefer backticks for this purpose personally. I'm open to switching if we come to a consensus though, not a big deal.

Reply:
Well, I decided I did not care enough to talk about it and just switched to single quotes.